lib/db/mysql.js

```diff
@@ -896,9 +896,20 @@ module.exports = function (log, error) {
   // Fields : uid, email, normalizedEmail, emailVerified, emailCode, kA, wrapWrapKb, verifierVersion, authSalt,
   //          verifierSetAt, createdAt, lockedAt, primaryEmail, profileChangedAt
   // Where  : emails.normalizedEmail = LOWER($1)
-  var GET_ACCOUNT_RECORD = 'CALL accountRecord_4(?)'
+  //
+  // There's a newer version of this query named `accountRecord_4`
+  // which pulls the `profileChangedAt` column from the database.
```

Comment:
Technically it doesn't pull `profileChangedAt` from db, it does `COALESCE(a.verifierSetAt, a.createdAt) AS profileChangedAt` as a workaround. `accountRecord_3` is the version that coalesced `a.profileChangeAt` with the above.

Reply:
Fair, I'll update the comment.